src/v/config/configuration.cc

```diff
@@ -1411,6 +1411,13 @@ configuration::configuration()
       "Enables rack-aware replica assignment",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       false)
+  , storage_strict_data_init(
+      *this,
+      "storage_strict_data_init",
+      "Requires that an empty file named `.redpanda_data_dir` be present in "
```

Comment:
Say a tiny bit more to explain the behavior "...in the data directory.  Redpanda will refuse to start if it is not found."  (it's not obvious what "requires" implies otherwise).

Reply:
Added in the additional explanation